_posts/2016-06-17-cynthia-intro.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Cynthia: introduction"
+date: 2016-06-17
+author: Cynthia Vint
+---
+
+#Cynthia Vint
```

Comment:
This line then becomes un-neccesary. The title gets auto-generated from the "title" field in the header.